condor_watch_q.py

```diff
@@ -473,18 +484,20 @@ def table_by_cluster_id(self):
                     if len(live_proc_ids) == 1:
                         x = "{}.{}".format(cluster_id, live_proc_ids[0])
                     else:
-                        x = "{}.{}-{}".format(
+                        x = "{}.{}".format(
                             cluster_id, min(live_proc_ids), max(live_proc_ids)
                         )
                     live_job_ids.append(x)
 
                 d[TOTAL] = sum(d.values())
                 d[CLUSTER_ID] = cluster_id
-                d[ACTIVE_JOBS] = ", ".join(live_job_ids)
+                x = datetime.datetime.now()
+                d[SUBMITTED] =x.strftime("%m")+"/"+x.strftime("%d")+" "+x.strftime("%H")+":"+x.strftime("%M")
```

Comment:
Please use whitespace spacing consistent with the rest of the file.  For example,

```suggestion
                d[SUBMITTED] = x.strftime("%m") + "/" + x.strftime("%d") + " " + x.strftime("%H") + ":" + x.strftime("%M")
```

Reply:
@JiaruFu I guess it doesn't say it anywhere, but just use https://pypi.org/project/black/ before committing and I'll be happy.